src/sass/_theme_variables.sass

```diff
@@ -59,7 +59,7 @@ $sidebar-border-color:                $table-border-color
 $sidebar-title-background-color:      $table-border-color
 
 // Sphinx highlight color
-$highlight-color:                     $yellow
+$highlight-color:                     rgba(255, 220, 0, 0.5)
```

Comment:
This is a different shade of yellow than the defined yellow from Wyrm. It would be better to drop the alpha channel and instead use a tint of the defined yellow from Wyrm. `rgb(248,225,135)` is a closer approximation of this.

Reply:
I don't follow you here.

I found Wyrm defining `$yellow` as `#F1C40F` which is `rgb(241,196,15)`.

Where `rgb(248,225,135)` comes from? Why it's better than my suggestion?

Reply:
`rgb(248,225,135)` is a lighter tint of `$yellow`. HSL is easier to understand than RGB here:

* `$yellow` == `rgb(241, 196, 15)` == `hsl(48.1, 89%, 50.2%)`
* light yellow == `hsl(48.1, 89%, 75%)` (we increased `l` (lightness) from 50% to 75%) == `rgb(248, 225, 135)`

Reply:
Niiiice! Thanks a lot of the explanation ;)